package.json

```diff
@@ -2,7 +2,20 @@
   "name": "ubie-ui",
   "version": "0.0.1",
   "description": "React components for creating Ubie applications.",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "README.md",
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "default": "./dist/index.js"
+    }
+  },
+  "sideEffects": false,
```

Comment:
The code included in the package will be without side effects.
Tree shaking is enabled with this setting.